src/plugin-slots/MobileHeaderSlot/index.jsx

```diff
@@ -11,6 +11,7 @@ const MobileHeaderSlot = ({
     slotOptions={{
       mergeProps: true,
     }}
+    pluginProps={props}
```

Comment:
pluginProps are the API for plugin slots, so they should only pass along necessary options conservatively.